src/csv.cr

```diff
@@ -280,15 +280,15 @@ class CSV
   # A negative index counts from the end.
   # Raises `IndexError` if any column doesn't exist
   # The behavior of returning a tuple is similar to `Hash#values_at`
-  def values_at(*columns : Int)
+  def values_at(*columns : Int) : Tuple(String, String)
```

Comment:
This make no sense at all. `*columns` can be any length, and be anything implementing `map`! The return type is definitely not a  `Tuple` in general.

Reply:
Whoops! Good catch, definitely stricter than it should be. In this case, we'd want to restrict it to a `Tuple(..., String,....)` of equivalent length as `*columns` gets passed in, but we don't have a way to communicate that at the moment.